docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+<!--
+Copyright 2023 The Khronos Group Inc.
+Copyright 2023 Valve Corporation
+Copyright 2023 LunarG, Inc.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+[![Khronos Vulkan][1]][2]
+
+[1]: https://vulkan.lunarg.com/img/Vulkan_100px_Dec16.png "https://www.khronos.org/vulkan/"
+[2]: https://www.khronos.org/vulkan/
+
+# Layers Configuration
+
+[![Creative Commons][3]][4]
+
+[3]: https://i.creativecommons.org/l/by-nd/4.0/88x31.png "Creative Commons License"
+[4]: https://creativecommons.org/licenses/by-nd/4.0/
+
+Vulkan supports intercepting or hooking API entry points via a layer framework. A layer can intercept all or any subset of Vulkan API entry points. Multiple layers can be chained together to cascade their functionality in the appearance of a single, larger layer.
+
+Vulkan layers give Vulkan application developers the ability to add additional functionality to applications without modifying the application itself, e.g., validating API usages, dumping API entry points or generating screenshots of specified frames.
+
+Vulkan layers can be configured using three different methods to match specific Vulkan developers workflows:
+- Using the Vulkan API: `VkCreateInstance()` and `VK_EXT_layer_settings`.
+- Using the `vk_layer_settings.txt` file, that can be generated by the GUI interface called *[Vulkan Configurator](https://vulkan.lunarg.com/doc/sdk/latest/windows/vkconfig.html)*.
+- Using environment variables.
+
+These three methods are implemented by the **Vulkan Layer Settings** library part of the *[Vulkan-Utility-Libraries](https://github.com/KhronosGroup/Vulkan-Utility-Libraries)* repository.
+Any layer project including this library will provide these three methods to control layer settings bringing consistency and easy of use of layers to the Vulkan community.
```

Comment:
> Any layer project including this library will provide these three methods

I read this as "Any layer project and also this library will provide..." which might be not the intention. 

Alternative: _Any layer project that uses this library_

Reply:
Fixed